built-in_analyses.js

```diff
@@ -37,17 +41,27 @@ function built_in_Irish(){
   //insert specified input to tree UI
   document.getElementById("stree-textarea").value = JSON.stringify(irish_trees);
   //exhaustivity options
-  document.getElementById("exhaustivityBox").setAttribute("checked", "checked");
-  document.getElementById("exhaustivityDetailRow").style.display = "block";
+  var exhaustivityBox = document.getElementById("exhaustivityBox")
+  exhaustivityBox.setAttribute("checked", "checked");
+  var exhaustivityDetail = exhaustivityBox.parentNode.parentNode.getElementsByTagName("td");
+  for (var x = 0; x < exhaustivityDetail.length; x++){
+    exhaustivityDetail[x].setAttribute("style", "display: table-cell");
+  }
+
+  //document.getElementById("exhaustivityDetailRow").style.display = "block";
   //some stuff for tones
-  var toneButtons = document.getElementsByName("toneOptions");
+  var toneCheckbox = document.getElementById("annotatedWithTones");
+  toneCheckbox.setAttribute("checked", "checked");
+  var toneButtons = toneCheckbox.parentNode.parentNode.getElementsByTagName("input");
   for(var x = 0; x < toneButtons.length; x++){
+    toneButtons[x].parentNode.setAttribute("style", "display: table-cell");
     if(toneButtons[x].value="addIrishTones_Elfner"){
-      toneButtons[x].setAttribute("checked", true);
+      toneButtons[x].setAttribute("checked", "checked");
```

Comment:
I think true should've been an okay value as well, but "checked" is fine too.

Reply:
I found that setAttribute() wasn't actually doing anything unless the second argument was a string. I could have used the string "true" instead of the value true, but the rest of the interface uses "checked" so I decided it would be good to keep that consistent.

Reply:
well there was a whole thing with setAttribute() so i've reverted back to toneButtons[x].checked = "checked";